TradingBot/Configuration/StrategyParametersConfiguration.cs

```diff
@@ -0,0 +1,45 @@
+﻿using TradingBot.Database.Entities;
+using TradingBot.Dto;
+
+namespace TradingBot.Configuration
+{
+    public sealed class StrategyParametersConfiguration
+    {
+        public required int MaxStocksBuyCount { get; init; }
+        public required int MinDaysDecreasing { get; init; }
+        public required int MinDaysIncreasing { get; init; }
+        public required decimal TopGrowingSymbolsBuyRatio { get; init; }
+
+        public static StrategyParametersConfiguration FromEntity(StrategyParametersEntity entity)
+        {
+            return new StrategyParametersConfiguration
+            {
+                MaxStocksBuyCount = entity.MaxStocksBuyCount,
+                MinDaysDecreasing = entity.MinDaysDecreasing,
+                MinDaysIncreasing = entity.MinDaysIncreasing,
+                TopGrowingSymbolsBuyRatio = (decimal)entity.TopGrowingSymbolsBuyRatio
```

Comment:
There's no point in having this be `decimal`. I used `decimal` with trading actions, assets, etc., because that's what Alpaca uses. But here it doesn't really help with anything.